[2024-01-28T14:52:20.166+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sample_dag_with_python_operator_v4.get_name scheduled__2024-01-19T00:00:00+00:00 [queued]>
[2024-01-28T14:52:20.179+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sample_dag_with_python_operator_v4.get_name scheduled__2024-01-19T00:00:00+00:00 [queued]>
[2024-01-28T14:52:20.180+0530] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-28T14:52:20.221+0530] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_name> on 2024-01-19 00:00:00+00:00
[2024-01-28T14:52:20.225+0530] {standard_task_runner.py:60} INFO - Started process 16307 to run task
[2024-01-28T14:52:20.237+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'sample_dag_with_python_operator_v4', 'get_name', 'scheduled__2024-01-19T00:00:00+00:00', '--job-id', '266', '--raw', '--subdir', 'DAGS_FOLDER/2-create-dag-with-py-opr.py', '--cfg-path', '/tmp/tmpu4fixm47']
[2024-01-28T14:52:20.240+0530] {standard_task_runner.py:88} INFO - Job 266: Subtask get_name
[2024-01-28T14:52:20.300+0530] {task_command.py:423} INFO - Running <TaskInstance: sample_dag_with_python_operator_v4.get_name scheduled__2024-01-19T00:00:00+00:00 [running]> on host mh1.myguest.virtualbox.org
[2024-01-28T14:52:20.453+0530] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='hdu' AIRFLOW_CTX_DAG_ID='sample_dag_with_python_operator_v4' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2024-01-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-19T00:00:00+00:00'
[2024-01-28T14:52:20.485+0530] {python.py:201} INFO - Done. Returned value was: None
[2024-01-28T14:52:20.494+0530] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=sample_dag_with_python_operator_v4, task_id=get_name, execution_date=20240119T000000, start_date=20240128T092220, end_date=20240128T092220
[2024-01-28T14:52:20.527+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-28T14:52:20.558+0530] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
